xml/ha_storage_protection.xml

```diff
@@ -869,25 +931,38 @@ SBD_WATCHDOG_TIMEOUT=5</screen>
    CGroup: /system.slice/sbd.service
            ├─<emphasis role="strong">1859 sbd: inquisitor</emphasis>
 [...]</screen>
-     <para>Use the process ID (in our example, <literal>1859</literal>) and
-      kill the SBD inquisitor:
-     </para>
-     <screen>&prompt.root;<command>kill</command> -9 1859 </screen>
+      </step>
+      <step>
+       <para>Simulate an SBD failure by killing the SBD inquisitor process.
+       In our example, the process ID of the SBD inquisitor is
+         <literal>1859</literal>):</para>
+       <screen>&prompt.root;<command>kill</command> -9 1859 </screen>
+       <para>
+        <remark>taroth 2018-04-28: @DEVs, What is the expected result?</remark>
+       </para>
+      </step>
+     </procedure>
     </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>Producing a Failure of a Resource Operation</term>
```

Comment:
How about "Producing a Failure of a Resource Operation that triggers fencing"

And I think it makes sense to  first mention the *typical* scenario where resource failure triggers fencing, for example:
"With a normal configuration, a failure of a resource stop operation triggers fencing. So you can somehow produce such a resource stop failure, if possible, to trigger fencing. If it's not easy to do so, another option is to temporarily change the configuration of a resource monitor operation and produce a monitor failure like the following to test that."

Reply:
Thanks, good idea. Integrated with https://github.com/SUSE/doc-sleha/commit/16c61998b.